sdk/eventhub/Azure.Messaging.EventHubs/tests/Primitives/EventProcessorTests.MainProcessingLoop.cs

```diff
@@ -0,0 +1,1422 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
Apologies to reviewers; had I thought a bit more, I probably would have remembered that the rename would trigger a full on "this is new" diff making it harder to understand the changes that were made.   In general, those towards the end of the file with names containing `StartProcessing` or `StopProcessing` and `Partition` were the focus area.